### DISCUSSION TITLE
Is there any way to make the response header use uppercase?

### DISCUSSION THREAD

**Discussion by Doslin** [tokio-rs/axum#2110] [Q&A] [answered]
Here's an example from actix-web

![image](https://github.com/tokio-rs/axum/assets/29807388/15019232-96e7-4038-bb6a-7eb273536538)

**Comment by davidpdrsn** [comment, marked as answer]
You don't need to post in multiple repos at once. That just spams the maintainers with emails.

But try https://docs.rs/hyper/latest/hyper/server/struct.Builder.html#method.http1_title_case_headers

**Comment by Doslin** [comment]
configure http1_title_case_headers in axum 0.7.x
ref:
https://github.com/tokio-rs/axum/blob/main/examples/serve-with-hyper/src/main.rs
https://docs.rs/hyper/latest/hyper/client/conn/http1/struct.Builder.html#method.title_case_headers
